recipe/meta.yaml

```diff
@@ -1,4 +1,9 @@
 {% set version = "2.7.3" %}
+{% set build = 2 %}
+{% set build_string = '' %}
+
+{% set build_string = 'mkl_' %}              # [linux64 and mkl_support == 'mkl']
+{% set build = build + 100 %}                # [linux64 and mkl_support == 'mkl']
```

Comment:
Should we removed changing the build number for `mkl` build? I can't find any documentation of how `track_feature` will interfere with build number - i.e which one will have priority. The closest documentation I have found on this topic is https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-channels.html?highlight=build%20number#managing-channels

Reply:
Both are fine to have and make the mkl build have less priority, but in two different ways.

Reply:
But here the mkl variant has higher build number, and I will expect that it get higher priority?

Reply:
Oops. We should change it and make mkl variant has lower build number.

Reply:
Done.